### PR TITLE
Add FAQs to support, request support, and docs pages

### PR DESCRIPTION
**Pull request by PatCapon39**
@neoformit I discussed this with @GarethPrice-Aus yesterday and we've gone with your suggested locations. Could you please double check that I've linked out to the faqs page correctly in each of the three places?

Once this is in dev and working, the FAQs page is all good to go onto main the next time dev is merged into main. Thanks!

### CODE REVIEW

**Comment by PatCapon39** [comment]
@neoformit as predicted, I did break the links. Should be good now

File: webapp/home/templates/home/requests/menu.html [hunk 1]
```diff
@@ -19,7 +19,7 @@ <h1> Galaxy Australia Support </h1>
         <h1 class="ga-blue"><span class="material-icons">help</span></h1>
         <div class="card-body">
           <p>
-            Need some assistance?
+            Checked <a href="/faqs.html">our FAQs</a> and still need some assistance?
             <br>
```
Comment (neoformit) [inline webapp/home/templates/home/requests/menu.html:23]: I would remove this <br> as the wrapping now looks a bit weird

**Review by neoformit — CHANGES_REQUESTED** [review]
One teeny issue and this is good to go, thanks Pat

File: webapp/home/templates/home/requests/menu.html (hunk 1 above)
Comment (PatCapon39) [reply webapp/home/templates/home/requests/menu.html]: Good call, done

**Comment by PatCapon39** [comment]
All fixed! (Didn't realise clicking resolve would make it disappear)

**Pull request merged**